wolox-books-angularjs/src/app/auth-guards/auth-guard.js

```diff
@@ -0,0 +1,19 @@
+const shouldAuth = path => {
+  return (path !== 'sign-up' && path !== 'login')
+}
+
+angular.module('app').run(['$rootScope', '$state', 'userService', function($rootScope, $state, userService) {
+    $rootScope.$on('$locationChangeStart', function(event, toState) {
+      let path = toState.slice(22);
+      if (shouldAuth(path) && !userService.isLoggedIn()) {
+        event.preventDefault();
```

Comment:
move this to the first line of the function

Reply:
This function should only prevent default actions when user's trying to go to unauthorized screens, otherwise it shouldn't intercede